native/common/include/jp_primitive_accessor.h

```diff
@@ -252,9 +269,9 @@ class JPConversionFloatWiden : public JPConversion
 {
 public:
 
-	virtual jvalue convert(JPJavaFrame *frame, JPClass *cls, PyObject *pyobj) override
+	virtual jvalue convert(JPMatch &match) override
 	{
-		JPValue *value = PyJPValue_getJavaSlot(pyobj);
+		JPValue *value = PyJPValue_getJavaSlot(match.object);
```

Comment:
This should be using the cache.  Please fix it, you slacker.

Reply:
:laughing: